packages/neuron-wallet/src/utils/lockUtils.ts

```diff
@@ -0,0 +1,96 @@
+import nodeService from '../startup/nodeService'
+import { CellOutPoint, OutPoint, Script } from '../appTypes/types'
+
+const { core } = nodeService
+
+export default class LockUtils {
+  static systemScriptInfo: { codeHash: string; outPoint: OutPoint } | undefined
+
+  static async systemScript() {
+    if (this.systemScriptInfo) {
+      return this.systemScriptInfo
+    }
+
+    const genesisHash: string = await core.rpc.getBlockHash('0')
+    const genesisBlock = await core.rpc.getBlock(genesisHash)
+    const systemScriptTx = genesisBlock.transactions[0]
+    const blake2b = core.utils.blake2b(32)
+    const systemScriptCell = systemScriptTx.outputs[0]
+    const { data } = systemScriptCell
+    if (typeof data === 'string') {
+      blake2b.update(core.utils.hexToBytes(data))
+    } else {
+      // if Uint8Array
+      blake2b.update(data)
+    }
+    const codeHash: string = blake2b.digest('hex')
+    const cellOutPoint: CellOutPoint = {
+      txHash: systemScriptTx.hash,
+      index: '0',
```

Comment:
If I recall correctly the index should be `1` instead?

Reply:
updated